chimera_app/shortcuts.py

```diff
@@ -231,6 +234,7 @@ def create_shortcut(entry, steam_shortcuts):
                 compat_data[compat_id] = {}
             compat_data[compat_id]['compat_tool'] = entry['compat_tool']
             if 'compat_config' in entry:
-                compat_data[compat_id]['compat_config'] = entry['compat_config']
+                (compat_data[compat_id]
```

Comment:
not a huge fan of this; I think it hurts readability

Reply:
Was done to improve linting mostly. But can be changed for better readability.